src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.ts

```diff
@@ -343,7 +343,7 @@ export class ServiceFormComponent extends CdForm implements OnInit {
       this.pools = resp;
     });
     this.cephServiceService.list().subscribe((services: CephServiceSpec[]) => {
-      this.services = services.filter((service: any) => service.service_type === 'rgw');
+      this.services = services;
```

Comment:
@avanthakkar But ingress services is not supported as backend for all services Is it?
So I feel it's good to show only what backend services is currently supported for ingress.

Reply:
Yeah, I've asked cephadm team for the list of services which has ingress support

Reply:
There should be a way to programatically do this. I was going to say that any daemon providing an HTTP service, but HAProxy supports non-HTTP too via TCP proxying (that's why NFS Ganesha can be HA'd here).

The following suggestion would need that services define their protocol (or alternative a `.supports_ingress`):
```suggestion
      this.services = services.filter((service: any) => ['http', 'https', 'tcp'].contains(service.service_proto));
```

BTW have you tested if this is enough for these new services, like the above mentioned nfs-ganesha?